app/src/main/java/br/com/catbag/gifreduxsample/asyncs/net/downloader/FileDownloader.java

```diff
@@ -31,9 +28,7 @@
     private static DownloadRoute sRoutes;
 
     public FileDownloader() {
-        if (sRoutes == null) {
-            sRoutes = new RoutesGenerator().createRoutes(DownloadRoute.class);
-        }
+        sRoutes = RetrofitBuilder.getInstance().createApiEndpoint(DownloadRoute.class, RiffsyRoutes.BASE_URL);
```

Comment:
Yeah. Actually this host URL have no impact in FileDownloader. Put this in constructor can suggests that BASE_URL is a important attribute fro FileDownloader. I've test pass an empty string, but errors happens.